README.md

```diff
@@ -388,6 +388,27 @@ $client = self::createPantherClient([
 * `PANTHER_FIREFOX_ARGUMENTS`: to customize Firefox arguments. You need to set `PANTHER_NO_HEADLESS` to fully customize.
 * `PANTHER_FIREFOX_BINARY`: to use another `firefox` binary
 
+### Changing the Size of the Browser Window
+
+This also controls the size of the screenshots.
+
+Chrome:
+
+```php
+$client = Client::createChromeClient(null, ['--window-size=1500,4000']);
+```
+or set it via `.env`:
+```
+PANTHER_CHROME_ARGUMENTS='--window-size=1500,4000'
+```
+Firefox:
+```php
+use Facebook\WebDriver\WebDriverDimension;
```

Comment:
Passing the `--window-size` argument to `createFirefoxClient()` should work isn't it?

Reply:
Hm, doing a quick check, this is not working for me:
```php
$client = Client::createFirefoxClient(null, ['--window-size=2500,4000']]);
```
At least the screenshot I'm getting is smaller (1280x955)

I can't remember exactly, but I guess that I did try this first, when I wrote the PR. If it would have been that easy, I wouldn't have dug out the `$client->manage()->window()->setSize($size);` way...

Reply:
found this while googling for the window size setting.
```
$size = new WebDriverDimension(1500, 2000);
$client->manage()->window()->setSize($size);
```

worked for me